bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -809,6 +809,28 @@ public static long win32_getHandle (Image image, int zoom) {
 	return image.getImageMetadata(zoom).handle;
 }
 
+/**
+ * @since 3.131
+ */
+public void drawOnGCAtTargetSize(Consumer<Long> drawFunction, int targetWidth, int targetHeight) {
```

Comment:
This should not be public but package protected as it is only supposed to be accessed from GC.
```suggestion
void drawOnGCAtTargetSize(Consumer<Long> drawFunction, int targetWidth, int targetHeight) {
```